packages/collector/test/tracing/protocols/graphql/schema.js

```diff
@@ -8,18 +8,18 @@ const graphQL = require('graphql');
 const graphqlSubscriptions = require('graphql-subscriptions');
 const data = require('./data');
 // Pino log spans are used to verify that follow up calls are traced correctly in a GraphQL entry.
-const pinoLogger = require('pino')();
+const bunyanLogger = require('bunyan').createLogger({ name: 'test-logger' });
```

Comment:
The reason for not using Winston here is that it is added to the list of packages allowed to load before the core module. In these situations, we might lose some spans that can cause the test to fail. We don't want to re-run the tests suppose this situation occurs